lib/iris/tests/unit/fileformats/pyke_rules/compiled_krb/fc_rules_cf_fc/test_build_auxiliary_coordinate.py

```diff
@@ -16,7 +16,7 @@
 # along with Iris.  If not, see <http://www.gnu.org/licenses/>.
 """
 Test function :func:`iris.fileformats._pyke_rules.compiled_krb.\
-fc_rules_cf_fc.build_auxilliary_coordinate`.
+fc_rules_cf_fc.build_dimension_coordinate`.
```

Comment:
I suspect the problem crept in because e2d856980ee8650fbd723c485accea77af53e4df has a merge conflict with the current master.